### PR TITLE
Allows to pass messages without object

### PR DESCRIPTION
**Pull request by felixLam**
This is an alternative to #17 allowing the sending of darwin notifications only by passing a nil message object.

Closes #17

### CODE REVIEW

File: Source/MMWormhole.m [hunk 1]
```diff
@@ -118,18 +118,21 @@ - (void)writeMessageObject:(id)messageObject toFileWithIdentifier:(NSString *)id
...
+    if (messageObject) {
+        NSData *data = [NSKeyedArchiver archivedDataWithRootObject:messageObject];
+        NSString *filePath = [self filePathForIdentifier:identifier];
+        
+        if (data == nil || filePath == nil) {
+            return;
+        }
+        
+        BOOL success = [data writeToFile:filePath atomically:YES];
+        if (!success) {
```
Comment (cnstoll) [inline Source/MMWormhole.m]: Can we add a blank line above the if for style consistency please? :)

Comment (felixLam) [inline Source/MMWormhole.m]: done

**Comment by cnstoll** [comment]
LGTM, just the one style comment. Thanks!

**Comment by cnstoll** [comment]
Merged. Thanks again!

**Pull request merged**